source/Sashimi.Terraform.Tests/TerraformValidatorFixture.cs

```diff
@@ -0,0 +1,251 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Autofac;
+using FluentAssertions;
+using FluentValidation.TestHelper;
+using NUnit.Framework;
+using Octopus.Server.Extensibility.HostServices.Model;
+using Sashimi.Server.Contracts;
+using Sashimi.Server.Contracts.ActionHandlers.Validation;
+using Sashimi.Server.Contracts.CloudTemplates;
+using Sashimi.Terraform.Validation;
+using Sashimi.Tests.Shared.Server;
+
+namespace Sashimi.Terraform.Tests
+{
+    [TestFixture]
+    public class TerraformValidatorFixture
+    {
+        TerraformValidator? validator;
+        
+        [SetUp]
+        public void Setup() {
+            var builder = new ContainerBuilder();
+            builder.RegisterModule<TerraformModule>();
+            builder.RegisterModule<ServerModule>();
+            using (var container = builder.Build())
+            {
+                validator = new TerraformValidator(container.Resolve<ICloudTemplateHandlerFactory>());
```

Comment:
Strictly speaking the container should be disposed at the end, otherwise any `IDisposable`s would be disposed by the time you try to use them?

Reply:
fixed